conf/messages.en

```diff
@@ -787,7 +788,7 @@ declaration.taricAdditionalCodes.header.hint = This will be the four digits that
 declaration.taricAdditionalCodes.header.tradeTariff.link = UK Trade Tariff
 declaration.taricAdditionalCodes.error.empty = Enter a TARIC code
 declaration.taricAdditionalCodes.error.invalid = Additional code must be 4 alpha numeric characters exactly
-declaration.taricAdditionalCodes.add.hint = Add TARIC code
+    declaration.taricAdditionalCodes.add.hint = Add TARIC code
```

Comment:
You have unnecessary Tab at the beginning of the line, PR is green so probably it's not a problem

Reply:
This has been addressed.